homeassistant/components/voip/assist_satellite.py

```diff
@@ -82,10 +83,11 @@ def async_add_device(device: VoIPDevice) -> None:
 class VoipAssistSatellite(VoIPEntity, AssistSatelliteEntity, RtpDatagramProtocol):
     """Assist satellite for VoIP devices."""
 
-    entity_description = AssistSatelliteEntityDescription(key="assist_satellite")
```

Comment:
There is no need to have an entity description to begin with.

Reply:
What should be done here then? https://github.com/home-assistant/core/blob/931c8f9e66193348fdcf92f93e7803d79b077f2f/homeassistant/components/voip/entity.py#L21